buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -221,6 +261,20 @@
         "retentionDays": "",
         "dropMetadataContainerName": "DropMetadata"
       }
+    },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Send Telemetry",
```

Comment:
It used to be called "MicroBuild Cleanup", I guess they changed the name of it.  All it says is that it should be run at the end of every MicroBuild build.